yarn.lock

```diff
@@ -3986,9 +4032,9 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "@hyperlane-xyz/hyperlane-token@workspace:typescript/token"
   dependencies:
-    "@hyperlane-xyz/core": 1.3.4
-    "@hyperlane-xyz/sdk": 1.3.4
-    "@hyperlane-xyz/utils": 1.3.4
+    "@hyperlane-xyz/core": 1.3.3
+    "@hyperlane-xyz/sdk": 1.3.3
+    "@hyperlane-xyz/utils": 1.3.3
```

Comment:
Why have these been downgraded? From the links of the PR I'd guess these yarn.lock changes were unintentional?